cfbs/index.py

```diff
@@ -291,3 +292,30 @@ def add(self, module, remote_config=None, dependent=None):
         else:
             print("Added module: %s" % module["name"])
         self.validate_added_module(module)
+
+
+class CFBSConfig:
```

Comment:
Maybe this should be a lazily initialized singleton class inheriting `CFBSJson`?